File: CentralComms.Data/Update.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Updates.Data
{
    public class Update
    {
        public int id { get; set; }
        public string title { get; set; }

        public string department { get; set; }
        public string updateType { get; set; }

        public string recipients { get; set; }

        public string message { get; set; }

        public string coverImage { get; set; }

        public string summary { get; set; }

        public string followUpType { get; set; }

        public string followUpAction { get; set; }

    }
}

File: angular-staff-web/src/app/update.ts
export class Update {
  id: number;
  title: String;
  department: string;
  recipients: string;
  typeOfUpdate: string;
  summary: string;
  message: string;
  scheduleMessage: Date;
  tags: string[];
  coverImage: string;
  followUpActionType: string;
  followUpAction: string;
  isPosted: boolean;

  constructor(
    title: string,
    department: string,
    recipients: string,
    typeOfUpdate: string,
    summary: string,
    message: string,
    scheduleMessage: Date,
    tags: string[],
    coverImage: string,
    followUpActionType: string,
    followUpAction: string
  ) {
    this.title = title;
    this.department = department;
    this.recipients = recipients;
    this.typeOfUpdate = typeOfUpdate;
    this.summary = summary;
    this.message = message;
    this.scheduleMessage = scheduleMessage;
    this.tags = tags;
    this.coverImage = coverImage;
    this.followUpActionType = followUpActionType;
    this.followUpAction = followUpAction;
    this.isPosted = false;
  }
}
